bessctl/commands.py

```diff
@@ -762,9 +762,9 @@ def run_file(cli, conf_file, env_map):
     _run_file(cli, os.path.expanduser(conf_file), env_map)
 
 
-@cmd('add worker WORKER_ID CORE', 'Create a worker')
-def add_worker(cli, wid, core):
-    cli.bess.add_worker(wid, core)
+@cmd('add worker WORKER_ID CORE [SCHEDULER]', 'Create a worker')
+def add_worker(cli, wid, core, scheduler='fast'):
```

Comment:
For a reason that I don't understand (I'm not great in python), specifying a default value like this doesn't work.

All the other functions here use a different pattern: default value=None and checking for None in the body of the function.  That seems to work